[quant][graphmode] Support quantization for `aten::append`

jerryzh168 opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #39645 [quant][graphmode] Support quantized::batch_norm2d_relu fusion for tracing
* **#39644 [quant][graphmode] Support quantization for `aten::append`**
* #39615 [quant][graphmode] Fix a corner case in handling `if` in insert_observers

Summary:

Test Plan:

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D21942112](https://our.internmc.facebook.com/intern/diff/D21942112)

jerryzh168 committed:
[quant][graphmode] Support quantization for `aten::append`

Summary:

Test Plan:

Reviewers:

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

raghuramank100 commented on torch/csrc/jit/passes/quantization/helper.cpp:
```diff
@@ -283,6 +283,9 @@ std::vector<Value*> getPassThroughInputs(Value* v) {
       inputs.push_back(v);
     }
     return inputs;
+  } else if (n->kind() == Symbol::aten("append")) {
```
How about adding a simple test to check for append?

jerryzh168 committed:
Update on "[quant][graphmode] Support quantization for `aten::append`"


Summary:

Test Plan:

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D21942112](https://our.internmc.facebook.com/intern/diff/D21942112)

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode] Support quantization for `aten::append`"


Summary:

Test Plan:

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D21942112](https://our.internmc.facebook.com/intern/diff/D21942112)

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode] Support quantization for `aten::append`"


Summary:

Test Plan:

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D21942112](https://our.internmc.facebook.com/intern/diff/D21942112)

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode] Support quantization for `aten::append`"


Summary:

Test Plan:

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D21942112](https://our.internmc.facebook.com/intern/diff/D21942112)

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode] Support quantization for `aten::append`"


Summary:

Test Plan:

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D21942112](https://our.internmc.facebook.com/intern/diff/D21942112)

[ghstack-poisoned]
